rlbot_gui/gui.py

```diff
@@ -198,14 +231,16 @@ def on_websocket_close(page, sockets):
 
 
 def is_chrome_installed():
-    return eel.browsers.chr.get_instance_path() is not None
+    return getattr(eel.browsers, "chm", getattr(eel.browsers, "chr", None)).get_instance_path() is not None
 
 
 def start():
     gui_folder = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'gui')
     eel.init(gui_folder)
 
-    options = {}
+    packet_reader = GameTickReader()
```

Comment:
If Rocket League is not running, the GUI will hang forever on startup waiting for the dll to be injected.

I'd very much like for the GUI to be able to open without any dependency on Rocket League or successful injection, both for the sake of the user experience, and also to avoid having every single bug related to those get blamed on the GUI.

Reply:
Any clue how to implement this better?